docs/development.md

```diff
@@ -24,6 +25,12 @@ cd $GOPATH/src/github.com/redhat-developer/odo
 git remote add upstream 'https://github.com/redhat-developer/odo'
 ```
 
+While cloning Odo, Windows terminal's such as PowerShell or CMD may throw error of `Filename too long`. To avoid such an error, set your Git config as so:
```

Comment:
Just realized grammar is incorrect, it should be: `While cloning Odo, the Windows terminal such as PowerShell or CMD may throw an error of `Filename too long`.